src/main/scala/epic/logo/ArgmaxInferencer.scala

```diff
@@ -0,0 +1,7 @@
+package epic.logo
+
+trait ArgmaxInferencer[T, Y, W] {
+
+  def argmax(weights : Weights[W], instance : T) : (Y, FeatureVector[W], Double)
```

Comment:
epic has a whole parallel infrastructure for this, but i'm gonna let it go through for now since we need it